modules/virt-configuring-obsolete-cpu-models.adoc

```diff
@@ -21,10 +21,11 @@ metadata:
   namespace: {CNVNamespace}
 spec:
   obsoleteCPUs:
-    cpuModels: <1>
+    cpuModels: <cpuModels>
```

Comment:
@SNiemann15, :slightly_smiling_face: is there any particular reason why you do not wish to use the AsciiDoc callouts like below?
 ```suggestion
    cpuModels: <cpuModels> # <1>
```

Reply:
Reverting to callouts after discussion with @ousleyp